boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

```diff
@@ -126,9 +126,9 @@
 
 &spi2 {
 	status = "ok";
-	sck-pin = <37>;
```

Comment:
Cześć! This change should be done in DTS overlay file belongs to the sample project. no need to change board DTS which might accidental impose badly other samples.

Reply:
I have added overlay.